llama.h

```diff
@@ -107,6 +107,20 @@ extern "C" {
         LLAMA_TOKEN_TYPE_BYTE         = 6,
     };
 
+    enum llama_token_attr {
+        LLAMA_TOKEN_ATTR_UNDEFINED    = 0,
+        LLAMA_TOKEN_ATTR_UNKNOWN      = 1 <<  1,
```

Comment:
Any reason for skipping the first bit here?

Reply:
I'm wondering why enums are being used to mask? Enums are constants, but this isn't right.